ArduPlane/Plane.cpp

```diff
@@ -1009,6 +1009,14 @@ bool Plane::is_taking_off() const
     return control_mode->is_taking_off();
 }
 
+bool Plane::start_takeoff(const float alt) {
+#if HAL_QUADPLANE_ENABLED
+    return quadplane.do_user_takeoff(alt);
```

Comment:
```suggestion
    return plane.quadplane.available() && quadplane.do_user_takeoff(alt);
```
`do_user_takeoff` doesn't check if quadplane is enabled. It might be a better fix to do the check in that function rather than here.

Reply:
Thanks. I applied here for now, we can discuss at dev call which class is responsible for checking if quadplane is available.